spec/server/update_model_spec.rb

```diff
@@ -74,31 +74,6 @@ def app
       Labors::Monster.attributes[:name] = original_attribute
     end
 
-    it "does not update attribute options with the incorrect data type" do
```

Comment:
This test isn't relevant anymore because it's not possible to set options with incorrect data types. `Sequel::Model` will cast values so they can be inserted in the database. This makes it easier to do things like insert a Ruby hash into a JSON column.

If we wanna do this kind of validation, we can look into `Sequel::Model`s type validation. https://sequel.jeremyevans.net/rdoc/files/doc/validations_rdoc.html#label-validates_type